target/linux/mediatek/image/filogic.mk

```diff
@@ -728,6 +728,23 @@ endif
 endef
 TARGET_DEVICES += xiaomi_redmi-router-ax6000-ubootmod
 
+define Device/zbtlink_zbt-z8102ax
+  DEVICE_VENDOR := Zbtlink
+  DEVICE_MODEL := ZBT-Z8102AX
```

Comment:
[It seems](https://whrl.pl/RgD6oT) there are two different models:
1. Z8102AX-M2;
2. Z8102AX-M2-T.

From what I've seen the outside differs significantly (e.g. 9 vs. 8 antennas).
Perhaps clarify?

Reply:
Could be different modem modules deployed...? But I have no idea.

Reply:
Hi, I just want to share my experience. Upon opening my ZBT-Z8103AX router, I was surprised to discover that one of the antennas, was not connected to the mainboard. The said antenna has a cable but it was taped to the plastic body of the router inside.

![image](https://github.com/openwrt/openwrt/assets/8255516/7058be6d-2a63-44ee-a083-a84149da2d73)

Maybe difference between the two is just a marketing gimmick (extra antenna).